components/teleport/Teleport.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <section>
+    <Loader v-model="isLoading" />
+    <b-field class="" label-position="inside" label="Select origin parachain">
+      <b-select
+        v-model="fromChain"
+        expanded
+        placeholder="Select parachain 1"
+        required>
+        <option v-for="chain in chains" :key="chain">{{ chain }}</option>
+      </b-select>
+    </b-field>
+
+    <b-field
+      class=""
+      label-position="inside"
+      label="Select destination parachain">
+      <b-select
+        v-model="toChain"
+        disabled
+        expanded
+        placeholder="Select parachain 2"
+        required>
+        <option v-for="chain in chains" :key="chain">{{ chain }}</option>
+      </b-select>
+    </b-field>
+
+    <b-field
+      class=""
+      label-position="inside"
+      :label="`Input recipient address (${toChain})`">
+      <b-input v-model="toAddress" expanded required></b-input>
+    </b-field>
+
+    <b-field class="" label-position="inside" label="Select currency">
+      <b-select
+        v-model="currency"
+        expanded
+        placeholder="Select currency"
+        required>
+        <option v-for="currency in currencies" :key="currency">
+          {{ currency }}
+        </option>
+      </b-select>
+    </b-field>
+
+    <b-field class="" label-position="inside" label="Input currency amount">
```

Comment:
BalanceInput cries in the corner

Reply:
BalanceInput 

<img src="https://media1.giphy.com/media/Yxg7MDkPj4kmI/giphy.gif"/>

Reply:
done